Svc/LinuxTimer/LinuxTimerComponentImplTaskDelay.cpp

```diff
@@ -19,7 +19,7 @@ namespace Svc {
 
   void LinuxTimerComponentImpl::startTimer(NATIVE_INT_TYPE interval) {
       while (true) {
-          Os::Task::delay(static_cast<Os::Task::ParamType>(interval));
+          Os::Task::delay(Fw::Time(static_cast<U32>(interval/1000), static_cast<U32>((interval*1000) % 1000000)));
```

Comment:
I think the second argument should be `static_cast<U32>(interval % 1000)`.

Reply:
agreed!

Reply:
Must multiply by 1000 somewhere to go from ms -> microseconds.